config/database.yml.example

```diff
@@ -14,10 +14,12 @@ production:
   adapter: mysql2
   encoding: utf8
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  host: <%= ENV.fetch("MYSQL_HOST") { '127.0.0.1' } %>
-  username: mapknitter
-  password: mapknitter
-  database: mapknitter
+  host: <%= ENV.fetch('DB_HOST') || 'db' %>
```

Comment:
I think here should be `ENV.fetch("MYSQL_HOST") { 'db' }`, that's a `env.fetch()` notation. It get `db` as the value for `MYSQL_HOST`, if not defined. Either this or `ENV.fetch("MYSQL_HOST", "db")` will work. https://ruby-doc.org/core-2.5.0/ENV.html#method-c-fetch